src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -22,6 +22,7 @@
 import seedu.address.model.person.Address;
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Name;
+import seedu.address.model.person.Note;
 import seedu.address.model.person.Person;
 import seedu.address.model.person.Phone;
 import seedu.address.model.person.Status;
```

Comment:
I can't comment on the correct line here but should we update the MESSAGE_USAGE string to include status and notes since they are included now too? So that when the user types in the wrong input it would show the correct usage format with the status and notes prefixes 🤔

Reply:
I've updated the help message here!
<img width="924" alt="image" src="https://user-images.githubusercontent.com/48595194/195383367-4d0089b6-beeb-4070-ab62-a308d0e1deef.png">